### ISSUE TITLE
Damage level estimates table showing wrong calculations

### ISSUE THREAD

**Issue by patrickkaleta** [clarity-h2020/csis#51]
CLARITY impact table at https://csis.myclimateservice.eu/study/1/step/22/view/table shows unplausible results

E.g., the sum of all “old” people (D1++  D5)  is ca. 100k for 28° and ca. 20k for 34°. The rest moved out? :-D

Haet wave 34°, 6 days
people
Age>65
number of people
19,389
18,550
7,099
1,358
135

Heat wave 28°, 6 days
people
Age>65
number of people
43,278
41,406
15,846
3,032
301

**Comment by bernhardsk** [comment]
Yes, they all moved out. It just was too hot for them. If you run the calculation for DC Sweden you will see that they are there now! ;)

This is due to the missing damage level probability functions (= vulnerability functions) from PLINIVS. We have/received only 1 set/table so far. Therefore we copied this 1 set multiple times in order to be able to calculate impact at all / also for other temperatures. I tried to call PLINIVS beginning of the week but couldn’t reach anybody. Therefore I will change the values to more realistic dummy values and restart the calculation which I explained in yesterdays telco. I can do this after today’s meeting at ZAMG.
